crates/ott-balancer/src/service.rs

```diff
@@ -76,11 +76,16 @@ impl Service<Request<IncomingBody>> for BalancerService {
             warn!("no route found for {}", req.uri().path());
             return Box::pin(async move { Ok(not_found()) });
         };
-        tracing::Span::current().record("handler", route.handler());
-        info!("inbound request");
+        let handler = **route.handler();
+        tracing::Span::current().record("handler", handler);
+        if handler == "status" || handler == "health" || handler == "metrics" {
+            debug!("inbound request");
+        } else {
+            info!("inbound request");
+        }
```

Comment:
this doesn't work because we use the tracing crate

Reply:
This doesn't work because `event!` expects a constant value for the log level. The exact error I got was "attempt to use a non-constant value in a constant".